ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarViewDemoController.swift

```diff
@@ -12,20 +12,25 @@ class AvatarViewDemoController: DemoController {
 
         createSection(withTitle: "Circle style for person", name: "Kat Larrson", image: UIImage(named: "avatar_kat_larsson")!, style: .circle)
         createSection(withTitle: "Square style for group", name: "NorthWind Traders", image: UIImage(named: "site")!, style: .square)
+        createSection(withTitle: "With image based frame", name: "Kat Larrson", image: UIImage(named: "avatar_kat_larsson")!, style: .circle, withFrame: true)
     }
 
-    private func createSection(withTitle title: String, name: String, image: UIImage, style: AvatarStyle) {
+    private func createSection(withTitle title: String, name: String, image: UIImage, style: AvatarStyle, withFrame: Bool = false) {
         addTitle(text: title)
         for size in AvatarSize.allCases.reversed() {
-            let imageAvatar = createAvatarView(size: size, name: name, image: image, style: style)
-            let initialsAvatar = createAvatarView(size: size, name: name, style: style)
+            let imageAvatar = createAvatarView(size: size, name: name, image: image, style: style, withFrame: withFrame)
+            let initialsAvatar = createAvatarView(size: size, name: name, style: style, withFrame: withFrame)
             addRow(text: size.description, items: [imageAvatar, initialsAvatar], textStyle: .footnote, textWidth: 100)
         }
         container.addArrangedSubview(UIView())
     }
 
-    private func createAvatarView(size: AvatarSize, name: String, image: UIImage? = nil, style: AvatarStyle) -> UIView {
+    private func createAvatarView(size: AvatarSize, name: String, image: UIImage? = nil, style: AvatarStyle, withFrame: Bool = false) -> UIView {
```

Comment:
this param name doesn't seem clear. frame make it sounds like user is setting the frame of the avatar or not using auto layout.